crates/wadm-types/src/lib.rs

```diff
@@ -247,9 +247,9 @@ pub struct SecretProperty {
     /// The name of the secret. This is used by a reference by the component or capability to
     /// get the secret value as a resource.
     pub name: String,
-    /// The source of the secret. This indicates how to retrieve the secret value from a secrets
+    /// The properties of the secret that indicate how to retrieve the secret value from a secrets
     /// backend and which backend to actually query.
-    pub source: SecretSourceProperty,
```

Comment:
@protochron let me know what you think here, looking at a manifest the `secrets` block was the only place where we had a different name for what is essentially a set of properties, so I figured renaming it here would be beneficial.